jmx_auth_test.py

```diff
@@ -55,6 +59,42 @@ def test_basic_auth(self):
         # superuser status applies to JMX authz too
         node.nodetool('-u cassandra -pw cassandra gossipinfo')
 
+    @since('4.1')
```

Comment:
That's because `nodetool invalidatejmxpermissionscache` has been introduced in 4.1 version.